Non-RPC/mumble-ping.py

```diff
@@ -23,19 +29,30 @@
     print(f"{time.time()}:NaN:NaN")
     sys.exit()
 
-print(f"recvd {len(data)} bytes")
+if verbose:
+    print(f"recvd {len(data)} bytes")
```

Comment:
Looks like this change not only introduces a format but also adds a verbose mode of operation, that is not invokable without changing the source?

Reply:
Thanks for pointing it out. Will add an optional `-v` flag.

Reply:
Added in 70c6ccc